Packs/Devo/Integrations/Devo_v2/Devo_v2.py

```diff
@@ -16,35 +17,31 @@
 from devodsconnector import error_checking
 from functools import partial
 
-''' GLOBAL VARS '''
-ALLOW_INSECURE = demisto.params().get('insecure', False)
-READER_ENDPOINT = demisto.params().get('reader_endpoint', None)
-READER_OAUTH_TOKEN = demisto.params().get('reader_oauth_token', None)
-WRITER_RELAY = demisto.params().get('writer_relay', None)
-WRITER_CREDENTIALS = demisto.params().get('writer_credentials', None)
-LINQ_LINK_BASE = demisto.params().get(
-    'linq_link_base', "https://us.devo.com/welcome")
-FETCH_INCIDENTS_FILTER = demisto.params().get('fetch_incidents_filters', None)
-FETCH_INCIDENTS_LIMIT = demisto.params().get('fetch_incidents_limit') or 50
-FETCH_INCIDENTS_DEDUPE = demisto.params().get(
-    'fetch_incidents_deduplication', None)
-TIMEOUT = demisto.params().get('timeout', '60')
-ITEMS_PER_PAGE = demisto.params().get('items_per_page', "1000")
-PORT = arg_to_number(demisto.params().get('port', '443') or '443')
-HEALTHCHECK_WRITER_RECORD = [{'hello': 'world', 'from': 'demisto-integration'}]
-HEALTHCHECK_WRITER_TABLE = 'test.keep.free'
-RANGE_PATTERN = re.compile('^[0-9]+ [a-zA-Z]+')
-TIMESTAMP_PATTERN = re.compile('^[0-9]+')
-TIMESTAMP_PATTERN_MILLI = re.compile('^[0-9]+.[0-9]+')
+
+""" GLOBAL VARS """
```

Comment:
since you keep the 'fetch_incidents_deduplication' parameter, please revert this line:
```
FETCH_INCIDENTS_DEDUPE = demisto.params().get(	
    "fetch_incidents_deduplication", None)
```

Reply:
I will, but it will include a comment of deprecation (for awareness).